.buildkite/pipeline-master.yml

```diff
@@ -19,6 +19,21 @@ steps:
           run: unit-test
           config: docker/buildkite/docker-compose.yml
 
+  - label: ":golangci-lint: validate code is clean"
+    agents:
+      queue: "workers"
+      docker: "*"
+    command: "./scripts/buildkite/golint.sh"
+    artifact_paths:
+      - ".build/golint/*.out"
```

Comment:
do we have any artifacts?

Reply:
Currently no. It's reserved for future usage.